content/en/istio-ca-certs-integrations/cert-manager-integration/_index.md

```diff
@@ -80,53 +84,46 @@ spec:
 EOF
 ```
 
-You can check the `issuers` using the `get issuers` command:
-
+Verify the "istio-ca" and "selfsigned" `issuers` are present and report `READY=True`:
 ```sh
-$ kubectl get issuers -n istio-system
-NAME         READY   AGE
-istio-ca     True    12m
+kubectl get issuers -n istio-system
 ```
 
-Once the Issuer has become ready,
+Once the `issuers` have become ready,
 [istio-csr](https://github.com/cert-manager/istio-csr) can be installed into the
 cluster.
 
+## Install istio-csr
+
 ```sh
 helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr
 ```
 
-Verify that istio-csr is running, and that the istiod Certificate is in a ready
-state.
-
+Verify the istio-csr deployment has successfully rolled-out.
 ```sh
-$ kubectl get pods -n cert-manager
-NAME                                       READY   STATUS    RESTARTS   AGE
-cert-manager-756bb56c5-cdvln               1/1     Running   0          111s
-cert-manager-cainjector-86bc6dc648-bjnrp   1/1     Running   0          111s
-cert-manager-istio-csr-5b9cd98696-v4f6j    1/1     Running   0          12s
-cert-manager-webhook-66b555bb5-4s7qb       1/1     Running   0          111s
-
-$ kubectl get certificates -n istio-system
-NAME       READY   SECRET       AGE
-istiod     True    istiod-tls   28s
+kubectl rollout status deploy/cert-manager-istio-csr -n cert-manager
 ```
 
-## Installing to a new Istio cluster
+Certificates "istio-ca" and "istiod" should be present and report `READY=True`.
+```sh
+kubectl get certificates -n istio-system
+```
 
-When installing a new Istio cluster, it must be configured to use istio-csr as
-the CA server by both the Istio control plane and workloads. The following
-configuration should be used when installing:
+## Install Istio
 
+Istio must be configured to use cert-manager as the CA server for both workload
+and Istio control plane components. The following configuration uses the
+IstioOperator resource to install Istio with cert-manager integration:
 ```sh
-cat << EOF > config.yaml
+getmesh istioctl install -y -f - <<EOF
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 metadata:
   namespace: istio-system
 spec:
   values:
     global:
+      # Change certificate provider to cert-manager istio agent for istio agent
```

Comment:
is this `Change certificate provider` an action that customer need to do, or a comment about what the value `caAddress` this is about? From the context, I think it's the latter?

Reply:
I clarified the comments.